tests/test_fast_scan.py

```diff
@@ -33,6 +33,35 @@ def test_PQ4_accuracy(self):
         assert recall_at_1 > 0.6
         # print(f'recall@1 = {recall_at_1:.3f}')
 
+    @unittest.skipUnless('AVX2' in faiss.get_compile_options(),
+            "only test while building with avx2")
```

Comment:
Could we have a test for the python-output of `faiss.get_compile_options()`?

I.e. `assert faiss.get_compile_options() in [<list of possible outputputs>]`?

Also, the indent here is off.